bin/setup_production_machine

```diff
@@ -22,6 +22,10 @@ rm -rf sr-build-tools &
 rm -rf ansible &
 wait
 
+# This is only necessary for non-desktop images
```

Comment:
True, but maybe having this would allow to test the script on a docker container?